### ISSUE TITLE
Issue with "<pre>: The Preformatted Text element": w3c Validator throwing error on <figure> example

### ISSUE THREAD

**Issue by 9Nish4** [mdn/content#12361]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre

#### What information was incorrect, unhelpful, or incomplete?
When I checked the code on the w3c validator. It throws an error "A figure element with a figcaption descendant must not have a role attribute."

#### Specific section or headline?
Validator error on adding role="img" on <figure>

#### Did you test this? If so, how?
By adding this code on Validator. Please let me know how to fix this and if we need to add role="img"

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/element/pre`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/pre/index.md
* Last commit: https://github.com/mdn/content/commit/e752a8fe731fbdb9061e39a6e2b797a7ccb7e2b5
* Document last modified: 2022-01-19T17:57:58.000Z

</details>

**Comment by himanshugarg** [comment]
I'm no expert in my field, but if this is anything to go by, yes it looks like we can remove the `role` attribute to satisfy the validator. Unless it appears in the example to satisfy some screen reader!

From - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role#description:
> most screenreaders will consider the element with role="img" set on it to be like a black box, and not access the individual elements inside it.

From: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure
![image](https://user-images.githubusercontent.com/2392803/151642940-fbe8b701-ede9-4253-be8c-8fd6837d8f01.png)
